components/brave_shields/core/browser/ad_block_component_service_manager.cc

```diff
@@ -34,6 +34,13 @@ namespace brave_shields {
 
 namespace {
 
+BASE_DECLARE_FEATURE(kAdBlockDefaultResourceUpdateInterval);
+constexpr base::FeatureParam<int> kComponentUpdateCheckIntervalMins{
+    &kAdBlockDefaultResourceUpdateInterval, "update_interval_mins", 100};
+BASE_FEATURE(kAdBlockDefaultResourceUpdateInterval,
+             "AdBlockDefaultResourceUpdateInterval",
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
you have `brave/components/brave_shields/core/common/features.h`. Please use it.